pkg/certificate/mrc_reconciler.go

```diff
@@ -80,6 +64,32 @@ func getSigningAndValidatingMRCs(mrcList []*v1alpha2.MeshRootCertificate) (*v1al
 	return mrcList[0], mrcList[1], nil
 }
 
+// ValidateMRCCombination takes a list of MRCs and ensures that the MRC combination is valid
+func ValidateMRCCombination(mrcList []*v1alpha2.MeshRootCertificate) error {
+	if len(mrcList) == 0 {
+		log.Error().Err(ErrNoMRCsFound).Msg("when handling MRC event, found no MRCs in OSM control plane namespace")
```

Comment:
This error message's context is too specific now that the logic has been extracted into its own function. Consider rewording